Tests/backend-specific-tests/Synapse/RunnerSynapseTest.php

```diff
@@ -151,4 +163,95 @@ public function testWorkspaceMapping()
+        $csv->writeRow(['test1', 'test1']);
+        $this->getClient()->createTableAsync('in.c-synapse-runner-test', 'mytable', $csv);
+        $this->getClient()->uploadFile(
+            $csv->getPath(),
+            (new FileUploadOptions())
+                ->setTags([self::ABS_TEST_FILE_TAG])
+                ->setFileName('abs-workspace-file.csv')
+        );
+        unset($csv);
+
+        $componentData = [
+            'id' => 'keboola.runner-workspace-abs-test',
+            'data' => [
+                'definition' => [
+                    'type' => 'aws-ecr',
+                    'uri' => '061240556736.dkr.ecr.us-east-1.amazonaws.com/keboola.runner-workspace-test',
+                    'tag' => 'latest',
+                ],
+                'staging_storage' => [
+                    'input' => 'workspace-abs',
+                    'output' => 'workspace-abs',
+                ],
+            ],
+            // https://keboola.slack.com/archives/C02C3GZUS/p1598942156005100
+            // https://github.com/microsoft/msphpsql/issues/400#issuecomment-481722255
+            'features' => ['container-root-user'],
+        ];
+
+        $configId = uniqid('runner-test-');
+        $components = new Components($this->client);
+        $configuration = new Configuration();
+        $configuration->setComponentId('keboola.runner-workspace-abs-test');
+        $configuration->setName('runner-tests');
+        $configuration->setConfigurationId($configId);
+        $components->addConfiguration($configuration);
+        $runner = $this->getRunner();
+
+        $runner->run(
+            $this->prepareJobDefinitions(
+                $componentData,
+                $configId,
+                [
+                    'storage' => [
+                        'input' => [
+                            'files' => [
+                                [
+                                    'tags' => [self::ABS_TEST_FILE_TAG],
+                                ],
+                            ],
+                        ],
+                    ],
+                    'parameters' => [
+                        'operation' => 'list-abs',
+                    ],
+                ],
+                []
+            ),
+            'run',
+            'run',
+            '1234567',
+            new NullUsageFile()
+        );
+        $options = new ListConfigurationWorkspacesOptions();
+        $options->setComponentId('keboola.runner-workspace-abs-test');
+        $options->setConfigurationId($configId);
+        self::assertCount(0, $components->listConfigurationWorkspaces($options));
+        $output = $this->getContainerHandler()->getRecords();
+        $blobFound = false;
+        foreach ($output as $blobName) {
+            if ($blobName === '/data/files/in/abs-workspace-file.csv') {
+                $blobFound = true;
+            } else {
+                echo "\nFound blob " . json_encode($blobName) . " which is not abs-workspace-file.csv\n";
```

Comment:
@odinuv this output `"\/tmp\/docker\/run-5fcf5048655848.32865018\/data\/in\/files\/213339795_run_5fcf503725f0c9.18705067\/run_5fcf503725f0c9.18705067"`. So the blob is there, but not sure best how to assert it

Reply:
it's not named properly